stix_shifter_modules/sentinelone/stix_translation/json/stix_2_1/to_stix_map.json

```diff
@@ -0,0 +1,641 @@
+    "object": "user"
+  },
+  "loginType": {
+    "key": "user-account.extensions.x-sentinelone-login-ext.login_type",
+    "object": "user"
+  },
+  "loginFailureReason": {
+    "key": "user-account.extensions.x-sentinelone-login-ext.login_failure_reason",
+    "object": "user"
+  },
+  "loginAccountDomain": {
+    "key": "user-account.extensions.x-sentinelone-login-ext.account_domain",
+    "object": "user"
+  },
+  "loginSessionId": {
+    "key": "user-account.extensions.x-sentinelone-login-ext.session_id",
+    "object": "user"
+  },
+  "tgtFileType": {
+    "key": "file.extensions.x-sentinelone-file-ext.file_type",
+    "object": "file"
+  },
+  "tgtFileExtension": {
+    "key": "file.extensions.x-sentinelone-file-ext.extention",
+    "object": "file"
+  },
+  "tgtFileId": {
+    "key": "file.extensions.x-sentinelone-file-ext.file_id",
+    "object": "file"
+  },
+  "tgtFileDescription": {
+    "key": "file.extensions.x-sentinelone-file-ext.description",
+    "object": "file"
+  },
+  "tgtFileLocation": {
+    "key": "file.extensions.x-sentinelone-file-ext.location",
+    "object": "file"
+  },
+  "tgtFileConvictedBy": {
+    "key": "file.extensions.x-sentinelone-file-ext.convicted_by",
+    "object": "file"
+  },
+  "tgtFileIsSigned": {
+    "key": "file.extensions.x-sentinelone-file-ext.is_signed",
+    "object": "file"
+  },
+  "registryPath": {
+    "key": "windows-registry-key.extensions.x-sentinelone-registry-ext.path",
+    "object": "registry_key"
+  },
+  "registryValueType": {
+    "key": "windows-registry-key.extensions.x-sentinelone-registry-ext.value_type",
+    "object": "registry_key"
+  },
+  "registryValueFullSize": {
+    "key": "windows-registry-key.extensions.x-sentinelone-registry-ext.full_size",
+    "object": "registry_key"
+  },
+  "registryOldValueType": {
+    "key": "windows-registry-key.extensions.x-sentinelone-registry-ext.old_value_type",
+    "object": "registry_key"
+  },
+  "registryOldValue": {
+    "key": "windows-registry-key.extensions.x-sentinelone-registry-ext.old_value",
+    "object": "registry_key"
+  },
+  "registryOldValueFullSize": {
+    "key": "windows-registry-key.extensions.x-sentinelone-registry-ext.old_value_full_size",
+    "object": "registry_key"
+  },
+  "endpointName": [{
+    "key": "x-oca-asset.hostname",
+    "object": "asset"
+  },
+  {
+    "key": "x-oca-event.host_ref",
+    "object": "event",
+    "references": "asset"
+  }],
+  "endpointOs": {
+    "key": "x-oca-asset.extensions.x-oca-endpoint-ext.endpoint_os",
+    "object": "asset"
+  },
+  "agentUuid": {
+    "key": "x-oca-asset.extensions.x-oca-endpoint-ext.agent_uuid",
+    "object": "asset"
+  },
+  "agentVersion": {
+    "key": "x-oca-asset.extensions.x-oca-endpoint-ext.agent_version",
+    "object": "asset"
+  },
+  "endpointMachineType": {
+    "key": "x-oca-asset.extensions.x-oca-endpoint-ext.machine_type",
+    "object": "asset"
+  },
+  "objectType": {
+    "key": "x-oca-event.object_type",
```

Comment:
This isn't part of the x-oca-event definition.

Reply:
Now we have mapped all to_stix custom object fields in from_stix_map, including object_type.